index.js

```diff
@@ -107,25 +107,25 @@ const startWithDevConfig = async appUrl => {
   } = require(`./${CONFIG_NAME}`)
   // test if local path configured
   if (packagePath && fs.existsSync(packagePath)) {
-    log.dev('load user provided package from fs:', packagePath)
+    log.dev('load user provided package from fs: ', packagePath)

```

Comment:
when logging content using `,`, it automatically adds a space. Opposed to the concatenate operator (`+`) that otherwise would glue those two expressions together.